src/taxonomy/TaxonomyListPage.test.jsx

```diff
@@ -118,4 +130,81 @@ describe('<TaxonomyListPage />', async () => {
     expect(mockDeleteTaxonomy).toBeCalledTimes(1);
     expect(mockSetToastMessage).toBeCalledWith(`"${taxonomies[0].name}" deleted`);
   });
+
+  it('should show all "All taxonomies", "Unassigned" and org names in taxonomy org filter', async () => {
+    useIsTaxonomyListDataLoaded.mockReturnValue(true);
+    useTaxonomyListDataResponse.mockReturnValue({
+      results: taxonomies,
+    });
+
+    const {
+      getByTestId,
+      getByText,
+      findByRole,
+      getAllByText,
+    } = render(<RootWrapper />);
+
+    await act(async () => {
+      expect(getByTestId('taxonomy-orgs-filter-selector')).toBeInTheDocument();
+      // Check that the default filter is set to 'All taxonomies' when page is loaded
+      expect(getByText('All taxonomies')).toBeInTheDocument();
+    });
+
+    // Open the taxonomies org filter select menu
+    const taxonomiesFilterSelectMenu = await findByRole('button', { name: 'All taxonomies' });
+    taxonomiesFilterSelectMenu.click();
```

Comment:
FYI, you can just use `getByRole` and directly do:

```suggestion
    fireEvent.click(getByRole('button', { name: 'All taxonomies' }));
```

Also instead of doing .click you should use `userEvent.click` or `fireEvent.click`. `fireEvent` will wrap the click in `act` so you don't need to do it yourself.